### ISSUE TITLE
Look into the research on kids, and come up with a more nuanced kids suggestion

### ISSUE THREAD

**Issue by catherio** [microcovid/microcovid#156]

**Comment by beshaya** [comment]
What about kids < 10?

Interesting gem hidden in Jimenez's spreadsheet: https://docs.google.com/spreadsheets/d/16K1OQkLD4BjgBdO8ePj6ytf-RpPMlJ6aXFg3PrIQBbQ/edit#gid=519189277 - 1st order approximation aerosols scale with body mass

**Comment by blanchardjeremy** [comment]
Ditto the question about kids < 10. Does that need its own issue?

**Comment by catherio** [comment]
Here's a resource we could use: https://mobile.twitter.com/daycareactivist/status/1318937254048444418

**Comment by catherio** [comment]
Another resource: https://www.nature.com/articles/d41586-020-02973-3

**Comment by joelchan** [comment]
Hi all, great work on this tool! There's a pretty good ongoing rapid review of evidence around children and COVID here: https://dontforgetthebubbles.com/evidence-summary-paediatric-covid-19-literature/

One recent systematic review and meta-analysis estimates a 44% reduction in risk of catching COVID for kids under 10-14yo compared to adults (but inconclusive re: transmission to others): https://jamanetwork.com/journals/jamapediatrics/fullarticle/2771181

Happy to help/collaborate with others to derive some estimates from the literature, maybe even consult with the authors of the rapid review? They are quite active on Twitter, might be game to get involved.

**Comment by beshaya** [comment]
Hi Joel,

We are actively looking for more folks to help intake research! If you'd
like to write up a summary of the research, in a similar style to the
whitepaper, we'd be excited to incorporate it.

**Comment by joelchan** [comment]
Great!! I’d be down to help draft something.

It’s personal for me too bc I reviewed the lit while deciding cost/benefit of opening our coop preschool (I have 5yo).

**Comment by joelchan** [comment]
@beshaya this would go under Q&A in the whitepaper, right? https://www.microcovid.org/paper/13-q-and-a

**Comment by blanchardjeremy** [comment]
@joelchan I think that's a fine place to draft it. 👍

**Comment by beshaya** [comment]
Yeah, let's put it into the Q&A section. If we end up with something
succinct and well supported, we can add it as an option in the calculator.

**Comment by catherio** [comment]
Another resource: https://pediatrics.aappublications.org/content/146/6/e2020027425

**Comment by catherio** [comment]
This somewhat supports the aforementioned ~44% reduction in catching COVID: https://services.aap.org/en/pages/2019-novel-coronavirus-covid-19-infections/children-and-covid-19-state-level-data-report/

kids are 12.4% of all cases in the US (and 22.6 percent of US people, so ~55%)

**Comment by catherio** [comment]
I also somewhat believe the body mass approximation, and kids <10 are about half the mass of adults (https://www.disabled-world.com/calculators-charts/height-weight-teens.php)

So I would support a Person Risk dropdown to modify "The people nearby are kids up to 10 years old" at 50%.

And an additional dropdown in Precautions "Age: Children are less likely to catch COVID-19. Are you calculating the risk to someone 10 or less years old?" to another 50% reduction.

**Comment by beshaya** [comment]
There's some good evidence in #606 suggests kids are half as likely to get
COVID from adults, but just as likely to give it to adults. Why? IDK.
Possibly parents are more likely to care for sick children (rather than
isolating the child). Possibly aerosols rise due to heat.

An important distinction is that a factor of 2 change in aerosol
production/consumption doesn't actually result in a factor of 2 difference
in chances of transmission! Behavior changes can easily make up that factor
of 2. Also, once we get past 1% transmission chances, probabilities don't
add (otherwise a person who generates 10x the average amount of aerosols
would have a 300% transmission risk, which is clearly nonsense).

**Comment by catherio** [comment]
oof, good point about isolating the child. ok, I'm now in favor of a 50% Precaution, but no reduction in outwards transmission.

**Comment by beshaya** [comment]
Reading more closely, the expected value from children is indeed lower than
from adults, but the 95% CI's overlap heavily. IDK how we want to handle
that.

**Comment by joelchan** [comment]
Circling back a bit: there's enough data on this now that there are 4 meta-analyses, all converging around a similar value of ~50% reduction in susceptibility (less likely to be infected, given exposure), but forward transmission is more murky. but we can reason that since we know asymptomatic/presymptomatic transmission is less efficient (can't recall by how much) than symptomatic transmission, and children are quite a bit more likely to be asymptomatic, there should be some reduction in forward transmission.

Meta-analyses (and a bunch more information) linked to in this Twitter thread, from a co-author on an ongoing lit review of COVID and children: https://twitter.com/apsmunro/status/1337426766806343687

**Comment by beshaya** [comment]
Yeah, I'd feel good with 50% Person Risk reduction for "is the
person/people at the event (a) child(ren)"

I'd also be ok with 50% reduction in Activity Risk for users who are
children, but I don't know how to explain this concisely - "Is the person
doing this activity under 10 years old?"

**Comment by joelchan** [comment]
I think reducing person risk (i.e., for the other people you're with) is a bit murkier (since if I understand correctly, data on reduced transmission *from* a child isn't quite as clear).

50% reduction if you're calculating for a user who is < 10yo seems quite fair. That's what I've been personally doing for my family (my son and daughter are 5 and 2). The way you phrased it seems quite reasonable! Not sure where it'd go in the UI though - it's not technically a precaution.

**Comment by nickybecky1** [comment]
> I think reducing person risk (i.e., for the other people you're with) is a bit murkier (since if I understand correctly, data on reduced transmission _from_ a child isn't quite as clear).
> 
If they're 50% less likely to catch it than adults, then they'd be less likely to spread in terms of just their risk profile going into the interaction though, right? Could it be that the option is to offer something like a child multiplier where the average (vaccinated) or average (unvaccinated) options are that would reduce the risk there? 

> 50% reduction if you're calculating for a user who is < 10yo seems quite fair. That's what I've been personally doing for my family (my son and daughter are 5 and 2). The way you phrased it seems quite reasonable! Not sure where it'd go in the UI though - it's not technically a precaution.

Could there be an option where the vaccines are essentially that says "unvaccinated but under 10 years old" that does that multiplying?

**Comment by nickybecky1** [comment]
> If they're 50% less likely to catch it than adults, then they'd be less likely to spread in terms of just their risk profile going into the interaction though, right? Could it be that the option is to offer something like a child multiplier where the average (vaccinated) or average (unvaccinated) options are that would reduce the risk there?
> 
> ETA: there seems to be a difference in risk levels for unvaccinated kids in families where all the adults are vaccinated vs unvaccinated kids in families where no one is vaccinated. Is there some way to incorporate that?

Okay - for now what I've done is entered our kiddo as having had 2 doses of AZ since that supposedly reduces their risk of infection by around 54%, which is close to what I was reading above about kid's infection risk, but I'm not sure if that underestimates their risk to others. 

I've also made a custom person "classmate"  who does outdoor hangs with other kiddos and has a household of 5 that has a fairly low risk tolerance which feels more accurate for what we're hearing about from other parents, and used that person for the classmates in the class.
